test/gtest/uct/ib/test_rc.cc

```diff
@@ -226,7 +243,30 @@ UCS_TEST_P(test_rc_iface_address, size_flush_remote)
     check_sizes(m_entity_flush_rkey, sizes);
 }
 
-UCT_INSTANTIATE_RC_DC_GGA_TEST_CASE(test_rc_iface_address)
+UCS_TEST_SKIP_COND_P(test_rc_iface_flush_remote, put_fence_no_flush_remote,
+                     !check_caps(UCT_IFACE_FLAG_PUT_ZCOPY))
+{
+    mapped_buffer sendbuf(64, 0ul, *m_e1);
+    mapped_buffer recvbuf(64, 0ul, *m_e2);
+    UCS_TEST_GET_BUFFER_IOV(iov, iovcnt, sendbuf.ptr(), 64, sendbuf.memh(),
+                            1);
+
+    uct_completion_t comp;
+    comp.func   = [](uct_completion_t*) {};
+    comp.count  = 1;
+    comp.status = UCS_OK;
+
+    // Trigger the use of atomic key, PUT fails with invalid atomic_mr_offset
+    ASSERT_UCS_OK(uct_ep_fence(m_e1->ep(0), 0));
```

Comment:
I'd set UCX_IB_PCI_RELAXED_ORDERING=y for the test to make sure it really tiggers usage of atomic key
or you can do some atomic op instead of put